src/main/java/duke/command/UnmarkCommand.java

```diff
@@ -0,0 +1,55 @@
+package duke.command;
+
+import duke.exception.CommandFormatException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import java.io.IOException;
+
+public class UnmarkCommand extends Command{
+    private String com;
+
+    /**
+     * Constructor for UnmarkCommand.
+     *
+     * @param i command in String format.
+     */
+    public UnmarkCommand(String i) {
+        super(0);
+        com = i;
+
+    }
+
+    @Override
+    public void execute(TaskList tL, Ui ui, Storage st) {
```

Comment:
Consider changing the name of this variable to something slightly clearer? Not completely sure what this variable is supposed to do.

Reply:
Okay, will rename it to be more specific.